apps/dashboard/app/models/launcher.rb

```diff
@@ -178,15 +178,21 @@ def update(params)
   end
 
   def submit(options)
-    adapter = adapter(options[:auto_batch_clusters]).job_adapter
+    cluster_id = if options.has_key?(:auto_batch_clusters)
+      options[:auto_batch_clusters]
+    else
+      smart_attributes.find { |sm| sm.id == 'auto_batch_clusters' }.value.to_sym
+    end
```

Comment:
Can you reformat this more like this?  I'm guessing we need this addition in the case that `auto_batch_clusters` is fixed?

```ruby
    cluster_id = if options.has_key?(:auto_batch_clusters)
                   options[:auto_batch_clusters]
                 else
                   smart_attributes.find { |sm| sm.id == 'auto_batch_clusters' }.value.to_sym
                 end
```